src/cromshell/utilities/HTTPUtils.py

```diff
@@ -0,0 +1,21 @@
+import tempfile
+import requests
+from cromshell.utilities import IOUtils
+
+
+def assertCanCommunicateWithServer(cromwell_server):
+    with tempfile.TemporaryFile() as f:
+        try:
+            curl_out = requests.get("{}/api/workflows/v1/backends".format(cromwell_server))
+        except ConnectionError:
+            print("Error: Cannot communicate with Cromwell server: {}".format(cromwell_server))
+            quit()
+        except requests.exceptions.RequestException as e:
+            print('exception caught', e)
```

Comment:
typically the error message you'd provide here is a bit more specific, so what's the intended exception you are capturing here?

Reply:
done, removed ambiguous exception